cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/etl/batch/BatchETLDBTest.java

```diff
@@ -142,15 +147,15 @@ private static void createTestTables(Connection conn) throws SQLException {
                      "BLOB_COL BLOB(100), " +
                      "CLOB_COL CLOB(100)" +
                      ")");
-      stmt.execute("CREATE TABLE my_dest_table AS (" +
-                     "SELECT * FROM my_table) WITH DATA");
+      stmt.execute("CREATE TABLE \"my_dest_table\" AS (" +
```

Comment:
HSQL by default (if you do not quote it), converts the table name to upper case. Quoting it retains the case. We need to think of whether we want to add the ``FieldCase``-kind of checks for table name too, but I'll leave that for later.